frontend/encore/vuejs.rst

```diff
@@ -34,6 +34,81 @@ also configure the `vue-loader options`_ via a callback:
         };
     });
 
+Building a simple Component
+---------------------------
+
+Once Vue is installed and ``webpack.config.js`` has been updated,
+you can build your first component and display a simple ``Hello World from Vue !``.
+
+In order to ease the process, the usage of ``.vue`` files is recommended,
+start by adding a ``main.js`` file inside the assets folder:
+
+.. code-block:: javascript
+
+    <!-- assets/main.js -->
+    import Vue from 'vue'
+    import Hello from './components/Hello.vue'
+
+    new Vue({
+        el: '#app',
+        render: h => h(Hello)
+    });
+
+This file's gonna call the component Hello from the components folder, once the component is found,
+Vue gonna instantiate a new Root component and attach your component to the HTML element who contain the app identifier.
+
+Now, let's build the Hello component :
+
+.. code-block:: html
+
+    <!-- assets/components/Hello.vue -->
+    <template>
+        <div id="app">
+            <p>{{ msg }}</p>
+        </div>
+    </template>
+
+    <script>
+        export default {
+            name: 'Hello',
+            data () {
+                return {
+                    msg: 'Hello World from Vue !'
+                }
+            }
+        }
+    </script>
+
+This component is pretty simple at this stage but once you're aware about Vue and his internal logic,
+you can easily integrate it into your own Twig views and logic.
+
+Now that your component is ready and that your Vue instance is configured,
+let's use Encore to build everything and call Vue inside your views,
+in order to ease the process, let's use the watcher :
```

Comment:
No space before dots

Reply:
We should try to improve the wording, too many "let's" in this sentence, I've got no suggestion right now.